File: server/FlightLog-server/src/main/java/bitc/fullstack503/flightlog/flightlogserver/dto/flightReserveMemberDTO.java
package bitc.fullstack503.flightlog.flightlogserver.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class flightReserveMemberDTO {
  @JsonProperty("AI")
  private int AI;

  @JsonProperty("flightPassport")
  private String flightPassport;

  @JsonProperty("flightReno")
  private String flightReno;

  @JsonProperty("flightUserId")
  private String flightUserId;

  @JsonProperty("flightMemFirstName")
  private String flightMemFirstName;

  @JsonProperty("flightMemLastName")
  private String flightMemLastName;

  @JsonProperty("flightMemStartSeatNum")
  private String flightMemStartSeatNum;

  @JsonProperty("flightStartPayCheck")
  private String flightStartPayCheck;

  @JsonProperty("flightMemLuggage")
  private String flightMemLuggage;

  @JsonProperty("flightMemArriveSeatNum")
  private String flightMemArriveSeatNum;
}

File: app/app/src/main/java/bitc/fullstack/FlightLog/dto/flightReserveMemberDTO.kt
package bitc.fullstack.FlightLog.dto

import com.google.gson.annotations.SerializedName

data class flightReserveMemberDTO(
  @SerializedName("AI")
  var AI: Int,

  @SerializedName("flightPassport")
  var flightPassport: String,

  @SerializedName("flightReno")
  var flightReno: String,

  @SerializedName("flightUserId")
  var flightUserId: String,

  @SerializedName("flightMemFirstName")
  var flightMemFirstName: String,

  @SerializedName("flightMemLastName")
  var flightMemLastName: String,

  @SerializedName("flightMemStartSeatNum")
  var flightMemStartSeatNum: String,

  @SerializedName("flightStartPayCheck")
  var flightStartPayCheck: String,

  @SerializedName("flightArrPayCheck")
  var flightArrPayCheck: String,

  @SerializedName("flightMemLuggage")
  var flightMemLuggage: String,

  @SerializedName("flightMemArriveSeatNum")
  var flightMemArriveSeatNum: String
)